x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/summary_step/components/post_save_options/post_save_options.tsx

```diff
@@ -73,34 +73,37 @@ export const PostSaveOptions: FC<Props> = ({ jobRunner }) => {
 
   return (
     <Fragment>
-      &emsp;
-      <EuiButton
-        isDisabled={
-          datafeedState === DATAFEED_STATE.STARTING || datafeedState === DATAFEED_STATE.STARTED
-        }
-        onClick={startJobInRealTime}
-        data-test-subj="mlButtonUseFullData3"
-      >
-        <FormattedMessage
-          id="xpack.ml.newJob.wizard.summaryStep.postSaveOptions.startJobInRealTime"
-          defaultMessage="Start job running in real time"
-        />
-      </EuiButton>
-      &emsp;
-      <EuiButton
-        isDisabled={
-          datafeedState === DATAFEED_STATE.STOPPED ||
-          datafeedState === DATAFEED_STATE.STARTING ||
-          watchCreated === true
-        }
-        onClick={() => setWatchFlyoutVisible(true)}
-        data-test-subj="mlButtonUseFullData"
-      >
-        <FormattedMessage
-          id="xpack.ml.newJob.wizard.summaryStep.postSaveOptions.createWatch"
-          defaultMessage="Create watch"
-        />
-      </EuiButton>
+      <EuiFlexItem grow={false}>
```

Comment:
Is that part of a `EuiFlexGroup` that's wrapping this component? Otherwise I'm wondering if `Fragment` should be replaced with `EuiFlexGroup`?

Reply:
this is embedded within a `EuiFlexGroup` here:
https://github.com/elastic/kibana/pull/44239/files#diff-77d690098d077c9044bf94fc5b995e82R144

Wrapping it in it's own `EuiFlexGroup` causes additional layout issues.